sdk/messaging/src/main/assets/bridge.js

```diff
@@ -46,6 +46,27 @@
                         WebViewBridge.postMessageToNative("close", {});
                     });
                 });
+
+                // add a listener to the message bridge init function, so we only listen to the bus if it's ready
+                window.addEventListener('inAppFormsInitMessage', function(event) {
+                    const supportedEventTypes = event.detail.eventTypes;
+                    console.log('Supporting message types:', supportedEventTypes);
+
+                    const messageBus = window['inAppFormsMessageBus']?.messageBus;
```

Comment:
TIL you can use `?` for optional chaining in JS nowadays

Reply:
Oof this is annoying, but this syntax may not be supporting in an iOS webview on our minimum supported iOS version, ios 13, see 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility